setup.py

```diff
@@ -187,7 +187,7 @@ def prepare_package(self):
                     shutil.copy(moltenvk_path,
                                 os.path.join(target, 'libMoltenVK.dylib'))
             else:
-                shutil.copy('runtimes/Release/taichi_core.dll',
+                shutil.copy('runtimes/taichi_core.dll',
```

Comment:
OOC do we need `Release` here (seems like we explicitly opened a Release folder above for ninja as well) ?

Reply:
The release folder is generated by msbuild. I can't think of a good way to detect whether msbuild or ninja is used. Maybe just enforce ninja?

Reply:
IIUC this line already implicitly enforced ninja? ;) If so we'll need to be extra careful and make sure the dev install is updated correctly. ;)

Reply:
Yes... kind of sad, but I can't think of a way to cleanly deal with this.

Reply:
@bobcao3 I don't mind switching to ninja for now ;) but this PR better stays consistent ;) . For example, `if (CMAKE_GENERATOR EQUAL "Ninja")` above kinda hints that we still support msbuild :D 
If we are sure ninja is easy to install on windows and we know how to update dev install doc, we could just error out on msbuild in cmake?